contrib/desktop-integration/README.md

```diff
@@ -1,11 +0,0 @@
-Desktop Integration
```

Comment:
nit: Maybe we can keep this README and add a link to https://github.com/jessfraz/dockerfiles/tree/master

Reply:
Yeah, I was considering that. Not sure how many people actually discover these (I think it'd be very low), so didn't think it was worth doing.